sys/cbor/cbor.c

```diff
@@ -319,10 +335,14 @@ static size_t decode_int(const cbor_stream_t *s, size_t offset, uint64_t *val)
 
     *val = 0; /* clear val first */
 
+    CBOR_ENSURE_SIZE_READ(s, offset + 1);
```

Comment:
is there a case where the `cbor_stream_t *s` can be NULL? 
The question also applies for all other cases where we pass a pointer to a macro.